src/libs/actions/IOU.js

```diff
@@ -65,6 +65,7 @@ function requestMoney(report, amount, currency, recipientEmail, participant, com
             iouReports[`${ONYXKEYS.COLLECTION.REPORT_IOUS}${chatReport.iouReportID}`],
             recipientEmail,
             amount,
+            currency,
```

Comment:
There are more usages of `IOUUtils.updateIOUOwnerAndTotal`. We'll need to update those too, e.g. https://github.com/Expensify/App/blob/2820c728c938d94b007aa747d4ba594547edc7af/src/libs/actions/IOU.js#L340-L344 and https://github.com/Expensify/App/blob/2820c728c938d94b007aa747d4ba594547edc7af/src/libs/actions/IOU.js#L572

Reply:
This is in `createSplitsAndOnyxData`, do you think it should have the same behavior?

Reply:
Yes, we still add the money request to the individual IOU report with each user.